axlearn/cloud/gcp/pathways_utils.py

```diff
@@ -246,6 +246,7 @@ def _build_pathways_head_container(self) -> dict:
         self._update_env_list(env_list, "JAX_PLATFORMS", "proxy")
         self._update_env_list(env_list, "ENABLE_PATHWAYS_PERSISTENCE", "1")
         self._update_env_list(env_list, "TPU_SKIP_MDS_QUERY", "true")
+        self._update_env_list(env_list, "TEST_UNDECLARED_OUTPUTS_DIR", "true")
```

Comment:
Can we add a comment here about its relationship with `IFRT_PROXY_USE_INSECURE_GRPC_CREDENTIALS` on proxy? So it is clear when we are ready to remove them. Thanks

Reply:
done